puppet/.puppet-lint.rc

```diff
@@ -0,0 +1 @@
+--no-autoloader_layout
```

Comment:
I think we should have all the puppet-lint flags either in `.puppet-lint.rc` or the `Rakefile`.. so maybe remove the flags from the Rakefile?